packages/core/src/visit-meta-css-dependencies.ts

```diff
@@ -19,10 +20,18 @@ export function visitMetaCSSDependenciesBFS(
         while (++index < items.length) {
             const imported = items[index];
             const res = resolver.resolveImported(imported, '');
+
             if (res?._kind === 'css' && !visited.has(res.meta.source)) {
                 visited.add(res.meta.source);
                 onMetaDependency(res.meta, imported, depth + 1);
                 q[depth + 1].push(...res.meta.getImportStatements());
+            } else if (res?._kind === 'js' && onJsDependency) {
+                const resolvedPath = resolver.resolvePath(imported.context, imported.request);
+
+                if (!visited.has(resolvedPath)) {
+                    visited.add(resolvedPath);
+                    onJsDependency(resolvedPath, imported);
```

Comment:
is there a limitation/issue when updating an imported module from a JS import (_CSS<-JS<-JS_)?

Reply:
Yep, we do not go deep inside the JS file, we might introduce this in feature.